microsetta_admin/templates/scan.html

```diff
@@ -247,16 +247,22 @@ <h4>Scan History</h4>
                                 <th class="bordered">Scan Timestamp</th>
                                 <th class="bordered">Sample Status</th>
                                 <th class="bordered">Technician Notes</th>
+                                <th class="bordered">Observations</th>
                             </tr>
                         </thead>
                         <tbody>
-                        {% for scan in scans_info %}
-                            <tr>
-                                <td class="bordered">{{ format_timestamp(scan['scan_timestamp']) }}</td>
-                                <td class="bordered">{{ scan['sample_status'] }}</td>
-                                <td class="bordered">{{ scan['technician_notes'] }}</td>
-                            </tr>
-                        {% endfor %}
+                            {% for scan in scans_info %}
+                                <tr>
+                                    <td class="bordered">{{ format_timestamp(scan['scan_timestamp']) }}</td>
+                                    <td class="bordered">{{ scan['sample_status'] }}</td>
+                                    <td class="bordered">{{ scan['technician_notes'] }}</td>
+                                    <td class="bordered"> 
+                                        {% for observation in scan['observations'] %}
```

Comment:
It would be good to render both the category and observation for any recorded observation on a given scan (e.g. "Tube: Insufficient ethanol" rather than "Insufficient ethanol". While the current observations are fairly clear, I can see future scenarios where the context of the category will be helpful.

![recorded_observations](https://github.com/biocore/microsetta-admin/assets/83246693/efa5ae38-d708-4fd2-af17-fa63fc6a532c)

Reply:
Added! There's an update to the API PR to make that possible.